include/wx/listbase.h

```diff
@@ -400,6 +400,12 @@ class WXDLLIMPEXP_CORE wxListCtrlBase : public wxSystemThemedControl<wxControl>
     bool InReportView() const { return HasFlag(wxLC_REPORT); }
     bool IsVirtual() const { return HasFlag(wxLC_VIRTUAL); }
 
+    // Check if the item is visible
+    bool IsVisible(long item);
```

Comment:
```suggestion
    virtual bool IsVisible(long item);
```

Reply:
@MaartenBent ,
Virtualizing is probably not needed, since it will be removed.

Reply:
The msw version overrides this function, so it needs to be virtual.

Reply:
You're right.
Fixing.